docs/source/tutorials_source/platform/tutorial_cropped_objects_metadata.py

```diff
@@ -0,0 +1,166 @@
+"""
+
+.. _lightly-tutorial-cropped-objects-metadata
+
+Tutorial 6: Find false negatives of object detection
+=============================================
+
+    In object detection applications, it can happen that the detector does not detect an object
+    because it did not see any examples of this or similar object yet.
+    This especially a problem in warehouse and retail applications,
+    as new products get added to the shelves every day. These new products with new appearence
+    often have a low so-called objectness score. Currently, finding these objects false detected
+    as non-objects needs a lot of manual labour, especially when having 1000s of new images coming
+    in every day. Lightly can make this work easier in a two-step approach:
+
+    1. A human finds one false negative and adds it to the missing examples.
+    2. Lightly finds all similar images which are also false negatives.
+    Thus it can propose to directly also add them as missing examples.
+
+    If there are e.g. 9 similar images for each missing example found by a human,
+    Lightly can speed up the process by a factor of 10.
+
+What you will learn
+--------------------
+
+* How to use an object detection model to crop objects out of a full image and save them as images.
+* How to save the object detection scores as metadata in a Lightly format.
+* How to upload the cropped images with their corresponding metadata to the webapp.
+* How to use the webapp to find false negatives and similar examples easily.
+
+Requirements
+-------------
+
+You can use your own dataset or the one we provide for this tutorial. The dataset
+we will use is the `SKU110k dataset <https://github.com/eg4000/SKU110K_CVPR19>` showing store shelves.
+The tutorial is computationally expensive if you run it on the full 110k images of products,
+thus we recommend running it on a subset of the dataset.
```

Comment:
Can you elaborate more on how you got the subset and how many images you ended up using? How long does it take to go through the tutorial? How did you download the dataset?

Reply:
Also see https://github.com/lightly-ai/lightly/pull/462#discussion_r686684825